remitano.gemspec

```diff
@@ -30,7 +30,7 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency("rspec-mocks", [">= 3.8"])
   spec.add_development_dependency("rdoc", ["~> 3.12"])
   spec.add_development_dependency("bundler")
-  spec.add_development_dependency("jeweler", ["~> 1.8.4"])
+  spec.add_development_dependency("jeweler", ["~> 2.0.0"])
```

Comment:
I update jeweler to 2.0.x version to be able to bundle this gem with bundler 2.x.